x-pack/plugins/observability_solution/infra/server/lib/alerting/inventory_metric_threshold/inventory_metric_threshold_executor.ts

```diff
@@ -64,11 +66,12 @@ export type InventoryMetricThresholdAlertContext = AlertContext; // no specific
 
 export type InventoryMetricThresholdAlert = Omit<
   ObservabilityMetricsAlert,
-  'kibana.alert.evaluation.values' | 'kibana.alert.evaluation.threshold'
+  'kibana.alert.evaluation.values' | 'kibana.alert.evaluation.threshold' | 'kibana.alert.group'
 > & {
   // Defining a custom type for this because the schema generation script doesn't allow explicit null values
   [ALERT_EVALUATION_VALUES]?: Array<number | null>;
   [ALERT_EVALUATION_THRESHOLD]?: Array<number | null>;
+  [ALERT_GROUP]?: Group[];
```

Comment:
do we need `ALERT_GROUP` in `InventoryMetricThresholdAlert` since we are not saving it in alert document?

Reply:
I had an issue with types since both inventory and metric threshold are saved in the same index; We need to have the same type for both. We don't save this field for inventory, but if we want to save it, it should have this type, so I thought it was OK to add this here.